notebooks/marimo/Exhibit_3_PSR.py

```
@@ -0,0 +1,255 @@
+    my_psr_z_T,
+    np,
+    pd,
+    ray,
+    stats,
+    tqdm,
+):
+    @ray.remote
+    def f2(rho, name, SR0_daily, SR1_daily):
+        if rho == 0:
+            _X0 = generate_non_gaussian_data(T_1, REPS, SR0=SR0_daily, name=name)
+            _X1 = generate_non_gaussian_data(T_1, REPS, SR0=_SR1_daily, name=name)
```

Ruff (0.15.10)

[error] Undefined name `_SR1_daily`

(F821)